test/functional/api/buffer_updates_spec.lua

```diff
@@ -1,4 +1,4 @@
-local helpers = require('test.functional.helpers')(after_each)
+local helpers = require('test.functional.helpers')()
```

Comment:
If we still need the `()` at the end, that's still kind of a hack. Do we need that?